bing_chilling_file.py

```diff
@@ -5,4 +5,4 @@
 
     print("BING CHILLING")
```

Comment:
```suggestion
    print("https://www.youtube.com/watch?v=dQw4w9WgXcQ")
```

Čia šiaip link'as turi būti

Reply:
asd